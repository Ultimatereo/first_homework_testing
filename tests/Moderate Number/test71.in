5
-8788467 4954447 -796528 -3089277 -9625717 -4289101
2763695 1287638 -4592286 -2871499 26397 6881704
4997932 1216402 107057 -8633564 -8241485 779306
5756839 4615815 -8838266 788787 -4246791 9300362
-7742965 797582 -315346 3071304 6102002 5362832
