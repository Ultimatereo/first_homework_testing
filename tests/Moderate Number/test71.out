0.965024
4.33079
0.812198
-0.806417
1.9852
